### ISSUE TITLE
module 'libagent.device.keepkey_defs' has no attribute 'get_public_node'

### ISSUE THREAD

**Issue by NTICompass** [romanz/trezor-agent#310]
I have a KeepKey, running firmware 5.10.2 (I'm not sure if I should update it), and I was trying to use `keepey-agent` for SSH.

Running `keepkey-agent nticompass@myServer` prompts me for my PIN then throws the following error:
```
2019-12-16 21:56:28,583 ERROR        close failed: 'KeepKeyClient' object has no attribute 'state'                                        [interface.py:134]
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/libagent/ssh/client.py", line 26, in export_public_keys
    pubkey = self.device.pubkey(identity=i)
  File "/usr/lib/python3.8/site-packages/libagent/device/keepkey.py", line 41, in pubkey
    return trezor.Trezor.pubkey(self, identity=identity, ecdh=ecdh)
  File "/usr/lib/python3.8/site-packages/libagent/device/trezor.py", line 82, in pubkey
    result = self._defs.get_public_node(
AttributeError: module 'libagent.device.keepkey_defs' has no attribute 'get_public_node'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/libagent/device/interface.py", line 132, in __exit__
    self.close()
  File "/usr/lib/python3.8/site-packages/libagent/device/trezor.py", line 73, in close
    self.__class__.cached_state = self.conn.state
AttributeError: 'KeepKeyClient' object has no attribute 'state'
Traceback (most recent call last):
  File "/usr/bin/keepkey-agent", line 10, in <module>
    sys.exit(ssh_agent())
  File "/usr/bin/keepkey_agent.py", line 5, in <lambda>
    ssh_agent = lambda: libagent.ssh.main(keepkey.KeepKey)
  File "/usr/lib/python3.8/site-packages/libagent/ssh/__init__.py", line 174, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/libagent/ssh/__init__.py", line 311, in main
    for pk in conn.public_keys():
  File "/usr/lib/python3.8/site-packages/libagent/ssh/__init__.py", line 210, in public_keys
    self.public_keys_cache = conn.export_public_keys(self.identities)
  File "/usr/lib/python3.8/site-packages/libagent/ssh/client.py", line 26, in export_public_keys
    pubkey = self.device.pubkey(identity=i)
  File "/usr/lib/python3.8/site-packages/libagent/device/keepkey.py", line 41, in pubkey
    return trezor.Trezor.pubkey(self, identity=identity, ecdh=ecdh)
  File "/usr/lib/python3.8/site-packages/libagent/device/trezor.py", line 82, in pubkey
    result = self._defs.get_public_node(
AttributeError: module 'libagent.device.keepkey_defs' has no attribute 'get_public_node'
```

I don't know if this is an issue in `keepkey-agent` or an issue in `keepkeylib`.

**Comment by romanz** [comment]
Thanks for reporting this issue, and sorry for the delayed response.
It's probably my fault - I changed the TREZOR's code and forgot to make sure it works also on KeepKey.
Could you please try older versions of `libagent` to see if they work?

**Comment by NTICompass** [comment]
@romanz Ok, I finally got around to testing this out for you.

Here's what I have (from `pip freeze`):
keepkey==6.3.1
keepkey-agent==0.9.0
libagent==0.13.1

With those versions, I get the error shown.  I tried downgrading to `libagent==0.13.0` and still got the same error.

Downgrading further to `libagent==0.12.1` seems to have fixed the problem.

**Comment by NTICompass** [comment]
Possible fix in PR #318, it seems to work for my device, but I don't know if I fixed it 100% correctly.
